WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -345,7 +345,7 @@ protected function check_argument_tokens( $context ) {
 
 		if ( empty( $tokens ) || 0 === count( $tokens ) ) {
 			$code = $this->string_to_errorcode( 'MissingArg' . ucfirst( $arg_name ) );
-			if ( 'domain' !== $arg_name || ! empty( $this->text_domain ) ) {
+			if ( ( 'domain' !== $arg_name || ! empty( $this->text_domain ) ) && ! in_array( 'default', $this->text_domain, true ) ) {
```

Comment:
Why use `in_array()` instead of a straight up string compare ?

Reply:
Because the `text_domain` is an array, like `array( 'customize-posts', 'default' )`.

Reply:
And so it is... :woman_facepalming: :tired_face: